test.sh

```diff
@@ -1,20 +0,0 @@
-EXIT_STATE=0
-
-pylint dash setup.py --rcfile=$PYLINTRC || EXIT_STATE=$?
-pylint tests -d all -e C0410,C0411,C0412,C0413,W0109 || EXIT_STATE=$?
-flake8 dash setup.py || EXIT_STATE=$?
-flake8 --ignore=E123,E126,E501,E722,E731,F401,F841,W503,W504 --exclude=metadata_test.py tests || EXIT_STATE=$?
-
-python -m unittest tests.development.test_base_component || EXIT_STATE=$?
-python -m unittest tests.development.test_component_loader || EXIT_STATE=$?
-python -m unittest tests.test_integration || EXIT_STATE=$?
-python -m unittest tests.test_resources || EXIT_STATE=$?
-python -m unittest tests.test_configs || EXIT_STATE=$?
```

Comment:
This script is no longer applicable, but at some point it may be nice to make some related script(s) for the new setup, to make it easy for people to run some or all of the CI tests locally. I guess now the regular tests are easy, just `pytest` right? But linting could use a helper.
Not necessary for this PR, but put it on a TODO somewhere.